src/main/java/net/mrscauthd/beyond_earth/events/Gravity.java

```diff
@@ -15,41 +15,45 @@ public static void Gravity(LivingEntity entity, GravityType type, Level world) {
         double venus = 0.04;
         double orbit = 0.02;
 
+        if (Methods.isWorld(world, Methods.overworld)) {
+            return;
+        }
+
         if (Methods.isWorld(world, Methods.moon)) {
             gravityMath(type, entity, moon, -2.5f);
         }
 
-        if (Methods.isWorld(world, Methods.mars)) {
+        else if (Methods.isWorld(world, Methods.mars)) {
             gravityMath(type, entity, mars, -2.0f);
         }
 
-        if (Methods.isWorld(world, Methods.glacio)) {
+        else if (Methods.isWorld(world, Methods.glacio)) {
             gravityMath(type, entity, mars, -2.0f);
         }
 
-        if (Methods.isWorld(world, Methods.mercury)) {
+        else if (Methods.isWorld(world, Methods.mercury)) {
             gravityMath(type, entity, mercury, -2.5f);
         }
 
-        if (Methods.isWorld(world, Methods.venus)) {
+        else if (Methods.isWorld(world, Methods.venus)) {
             gravityMath(type, entity, venus, -2.0f);
         }
 
-        if (Methods.isOrbitWorld(world)) {
+        else if (Methods.isOrbitWorld(world)) {
             gravityMath(type, entity, orbit, -2.5f);
         }
     }
 
     public enum GravityType {
-        PLAYER,LIVING
+        PLAYER, LIVING
     }
 
     public static boolean playerGravityCheck(Player player) {
         return !player.getAbilities().flying && !player.isFallFlying() && !player.isInWater() && !player.isInLava() && !player.isNoGravity() && !player.hasEffect(MobEffects.SLOW_FALLING) && !player.hasEffect(MobEffects.LEVITATION);
     }
 
     public static boolean livingGravityCheck(LivingEntity entity) {
-        return !entity.isFallFlying() && !entity.isInWater() && !entity.isInLava() && !entity.isNoGravity() && !(entity instanceof Player) && !Methods.AllVehiclesOr(entity) && !entity.hasEffect(MobEffects.SLOW_FALLING) && !entity.hasEffect(MobEffects.LEVITATION);
+        return !(entity instanceof Player) && !entity.isFallFlying() && !entity.isInWater() && !entity.isInLava() && !entity.isNoGravity() && !Methods.AllVehiclesOr(entity) && !entity.hasEffect(MobEffects.SLOW_FALLING) && !entity.hasEffect(MobEffects.LEVITATION);
```

Comment:
I moved the `entity instanceof Player` check to the start as it is faster than the other checks and will avoid doing the other earlier checks that come after it if unnecessary